pages/_campus/shuttle-factories/index.vue

```diff
@@ -77,8 +77,9 @@ export default {
     searchFilterMixin(),
     updateListMixin(SHUTTLE_FACTORIES, {
       mask: DEFAULT_MASK,
-      customList: async (l, { params }) => {
-        const list = await l.setFilter('campus', params.campus);
+      customQuery: (q, { params }) => q.setCampus(params.campus),
+      customList: async (l) => {
+        const list = await l.callback();
```

Comment:
```suggestion
        const list = await l;
```

Same thing here, probably not expected to call `callback`directly